frontends/web/src/routes/buy/info.tsx

```diff
@@ -195,9 +195,8 @@ class BuyInfo extends Component<Props, State> {
                                     <h1 class={style.title}>{t('buy.title', { name })}</h1>
                                     <Select
                                         label={t('buy.info.selectLabel')}
-                                        placeholder={t('buy.info.selectPlaceholder')}
                                         options={[{
-                                                text: t('buy.info.selectPlaceholder'),
+                                                text: t('buy.info.selectLabel'),
```

Comment:
@benma so the label will say: "Choose your account"
and the select placeholder also "Choose your account"
is that what you really really want?

Reply:
good point - removed the label as well :D

Reply:
i went back and now show `Select a coin` only. I feel it's more intuitive, you want to buy a coin, not an account.

Reply:
sorry I changed it to "select an account" again :P 
let me know if I should redo.
lets discusss in chat ping @jadzeidan